src/main/java/com/redhat/cloud/notifications/NotificationsApp.java

```diff
@@ -5,17 +5,33 @@
 
 import javax.enterprise.event.Observes;
 
+import java.util.regex.Pattern;
+
 public class NotificationsApp {
+
     private static final String BUILD_COMMIT_ENV_NAME = "OPENSHIFT_BUILD_COMMIT";
     private static final String BUILD_REFERENCE_ENV_NAME = "OPENSHIFT_BUILD_REFERENCE";
     private static final String BUILD_NAME_ENV_NAME = "OPENSHIFT_BUILD_NAME";
 
+    public static final String REGEX_FILTER = ".*(/health|/metrics) HTTP/[0-9].[0-9]\\\" 200.*";
+
     private static final Logger LOG = Logger.getLogger(NotificationsApp.class);
 
+    // we do need a event as parameter here, otherwise the init method won't get called.
     void init(@Observes StartupEvent ev) {
+        filterAccessLogs();
+
         showVersionInfo();
     }
 
+    private void filterAccessLogs() {
+        java.util.logging.Logger accessLog = java.util.logging.Logger.getLogger("access_log");
+        accessLog.setFilter(record -> {
+            final String logMessage = record.getMessage().trim();
```

Comment:
Is `trim()` really necessary? If possible, it would be better for performances to remove it.

Reply:
At the moment it is. I don't know how to change the regex to get rid of the trim() call.

Reply:
What makes it necessary? The current regex starts and ends with `.*` so it should match even with leading or trailing spaces.

Reply:
The `\n` at the end makes it necessary, as `.*` only matches until end of line, but not the `\n`.

Reply:
Oh yes, I forgot about that. We could use the [DOTALL](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html#DOTALL) flag.

Reply:
I tried the DOTALL in Pattern.compile(FILTER_REGEX); already. It didn't work..

Reply:
Heiko fixed it by doing fancy stuff :-D thanks :)